src/neondis.cpp

```diff
@@ -0,0 +1,41 @@
+#include <fstream>
+#include <iostream>
+#include <sstream>
+
+#include "disassembler.h"
+
+int main(int argc, char *argv[])
+{
+    if (argc < 2) {
+        fprintf(stderr, "Usage: %s file1.neonx file2.neonx...fileN.neonx\n", argv[0]);
+        exit(1);
+    }
+
+    int a = 1;
+    while (a < argc) {
+        std::cout << "Disassembeling " << argv[a] << "...\n";
+
+        const std::string name = argv[a];
+
+        std::ifstream inf(name, std::ios::binary);
+        std::stringstream buf;
+        buf << inf.rdbuf();
+
+        // ToDo: Add Script header for better byte code detection.
+        if (name[name.length()-1] != 'x') {
+            fprintf(stderr, "%s is not a neonx file.\n", argv[a]);
+            ++a;
+            continue;
+        }
+
+        std::vector<unsigned char> bytecode;
+        std::string s = buf.str();
+        std::copy(s.begin(), s.end(), std::back_inserter(bytecode));
+
+        // ToDo: Detect Debug info stored in Bytecode, and pass pointer to it.
+        disassemble(bytecode, std::cerr, NULL);
```

Comment:
Since disassembly is the primary purpose of this program, the output should go to stdout, not stderr.